react/Geolocation.js

```diff
@@ -129,32 +161,31 @@ class Geolocation extends Component {
         this.setState({ permissionStatus: DENIED })
         searchPickupAddressByGeolocationEvent({
           deniedGeolocation: true,
+          elapsedTime,
         })
         break
       case 2: // POSITION_UNAVAILABLE
         setAskForGeolocation(false)
         setGeolocationStatus(ERROR_NOT_FOUND)
```

Comment:
Why this isn't doesn't change with the line below? Asking because I'm not sure which is which in the UI 🤔

Reply:
@wdsrocha not sure if I understood the question properly, but:

`setGeolocationStatus` sets, among other things, _which error message is going to appear_, if any. The current status seems correct, and is being handled here: https://github.com/vtex/pickup-points-modal/pull/172/files/744fab837acff142baf96a6a2483fbbbe534af6d#diff-9d722fef23968cf4329d7c6464686f52a6e5458b553543268587b8bd8e526388R18-R24

`setCurrentActiveState` sets, among other things, which "page" the modal will display, whether the search field, the map, the error message, etc. This one was deemed incorrect, and was changed. Then which error message it will display along with the search field is set by the current "geolocation status"